tables/isolate_mutations.csv

```diff
@@ -12954,3 +12954,16 @@ BA.1 RBD,S,496,S,NULL,NULL
 BA.1 RBD,S,498,R,NULL,NULL
 BA.1 RBD,S,501,Y,NULL,NULL
 BA.1 RBD,S,505,H,NULL,NULL
+B.1.351 Spike:-18F+241del-244del-256I+655Y+1027I+1176F,S,80,A,NULL,NULL
+B.1.351 Spike:-18F+241del-244del-256I+655Y+1027I+1176F,S,215,G,NULL,NULL
+B.1.351 Spike:-18F+241del-244del-256I+655Y+1027I+1176F,S,241,del,NULL,NULL
+B.1.351 Spike:-18F+241del-244del-256I+655Y+1027I+1176F,S,242,del,NULL,NULL
+B.1.351 Spike:-18F+241del-244del-256I+655Y+1027I+1176F,S,243,del,NULL,NULL
+B.1.351 Spike:-18F+241del-244del-256I+655Y+1027I+1176F,S,417,N,NULL,NULL
+B.1.351 Spike:-18F+241del-244del-256I+655Y+1027I+1176F,S,484,K,NULL,NULL
+B.1.351 Spike:-18F+241del-244del-256I+655Y+1027I+1176F,S,501,Y,NULL,NULL
+B.1.351 Spike:-18F+241del-244del-256I+655Y+1027I+1176F,S,614,G,NULL,NULL
+B.1.351 Spike:-18F+241del-244del-256I+655Y+1027I+1176F,S,655,Y,NULL,NULL
+B.1.351 Spike:-18F+241del-244del-256I+655Y+1027I+1176F,S,701,V,NULL,NULL
+B.1.351 Spike:-18F+241del-244del-256I+655Y+1027I+1176F,S,1027,I,NULL,NULL

```

Comment:
I didn't see 1027I and 1176F in Beta from table 1. However they are both in Gamma. Is this a typo?

Reply:
Yes, I think that was a typo

Reply:
I also edited the mutation list for lambda, it has an additional 253del